tests.sh

```diff
@@ -56,7 +56,7 @@ if [[ "$1" == "integration" ]]; then
   PING_LOOP_PID=$!
 
   # Run integration tests, redirect output to tmp file
-  ./gradlew check -x test &> ${BUILD_OUTPUT}
+  ./gradlew check -x test --debug &> ${BUILD_OUTPUT}
```

Comment:
Don't forget to drop these changes (at least this one, I think the 1000 lines one is fine).